drivers/hmis/app/models/hmis/filter/enrollment_filter.rb

```diff
@@ -49,4 +50,54 @@ def with_project_types(scope)
   def with_search_term(scope)
     with_filter(scope, :search_term) { scope.matching_search_term(input.search_term) }
   end
+
+  def with_household_tasks(scope)
+    with_filter(scope, :household_tasks) do
+      return scope.all unless input.household_tasks&.include?('ANNUAL_DUE')
+
+      # Start of the 60-day window during which this year's Annual should be performed. (Anniversary - 30 days)
+      start_date = Arel.sql <<~SQL
+        make_date(
+          extract(year from current_date)::integer,
+          extract(month from "EntryDate")::integer,
+          1
+        ) - interval '30 days' + "interval"((extract(day from "EntryDate")::integer - 1) || ' days')
```

Comment:
Can you add a comment here about leap year? thanks!

Reply:
Yup done. It's a bit wordy, so let me know if you want me to change it to be more concise